test-clients/src/main/java/com/hedera/services/bdd/spec/HapiApiSpec.java

```diff
@@ -133,6 +140,30 @@ public void incrementNumLedgerOps() {
 		ledgerOpCountCallbacks.stream().forEach(c -> c.accept(newNumLedgerOps));
 	}
 
+	public synchronized void saveSingleAccountBalances(SingleAccountBalances sab) {
+		accountBalances.add(sab);
+	}
+
+
+	public void exportAccountBalances(Supplier<String> dir) {
+		Instant now = Instant.now();
```

Comment:
If this file won't include the actual consensus timestamp, I'd suggest leaving the `consensusTimestamp` field empty just so it doesn't become confusing. 😀

Reply:
Make senses. I will do that.